Base.rte/Devices/Shared/Scripts/MuzzleSmoke.lua

```diff
@@ -0,0 +1,21 @@
+function Create(self)
```

Comment:
Yeah, I removed the extras because they were mostly the same script with different values, with the exception of one script that was supposed to be on the bullet rather than the firearm (and it was named in a similar fashion to the others, which was misleading af)

Might need a converter tag for the removed scripts! @MyNameIsTrez

Reply:
Okay, figured it was something like that. If Trez doesn't reply here, please let him know in discord (once the branch is merged) so he doesn't miss this change,

Reply:
Got it, thanks.